### PR TITLE
Add a new method for removeduplicates!

### PR DESCRIPTION
**Pull request by amellnik**
Which takes an array of column names rather than a single string.  Added
a deprecation warning to the old version.  This should resolve https://github.com/JuliaDB/SQLite.jl/issues/94.

### CODE REVIEW

**Comment by amellnik** [comment]
I'm not too familiar with the build logs, but it looks like the failing on is due to some issues with Coveralls on 0.5 and not with this module?  Can anyone confirm this or let me know if there's some other issue, I haven't started using the 0.5 previews yet.

**Comment by Sean1708** [comment]
Thanks for the work @amellnik.

Yeah the failure looks like a coveralls issue to me, plus the tests run fine on my macbook. 

The patch looks good, @quinnj I'm happy for this to merge if you are.

**Comment by Sean1708** [comment]
Thinking about it, we should probably add a test case for this as well.

**Comment by amellnik** [comment]
Good point, I added a standalone test that uses the new method.

File: test/runtests.jl [hunk 1]
```diff
@@ -290,3 +290,24 @@ dt2 = SQLite.query(db, "Select * from temp")
...
+SQLite.drop!(db, "temp", ifexists=true)
+sink = SQLite.Sink(dt, db, "temp")
+Data.stream!(dt, sink)
+SQLite.removeduplicates!(db, "temp", ["ints","strs"]) #New format
+dt3 = SQLite.query(db, "Select * from temp")
+@test get(dt3[1,1]) == 1
+@test get(dt3[1,2]) == "A"
+@test get(dt3[2,1]) == 2
+@test get(dt3[2,2]) == "B"
+@test get(dt3[3,1]) == 2
```
Comment (Sean1708) [inline test/runtests.jl]: Is this correct? Shouldn't it be `@test get(dt3[3,1]) == 3`?

Comment (amellnik) [inline test/runtests.jl]: I believe it is -- the pairs to start are 

```
1 A
1 A
2 B
2 C
3 C
```

removing duplicates with both columns as the key for uniqueness drops only the 2nd row.

Comment (Sean1708) [inline test/runtests.jl]: You're right, I thought there were two `B`s.

**Comment by amellnik** [comment]
Gentle ping to @quinnj and @Sean1708 -- does this look ok to merge?

**Comment by Sean1708** [comment]
Travis failure is unrelated, so I'm happy for this to merge. I'll wait for @quinnj to give his input first though.

**Comment by quinnj** [comment]
Sorry for the delay! Thanks!

**Pull request merged**